README.md

```diff
@@ -43,22 +43,20 @@ conda install -c conda-forge openjdk=21 maven -y
 pip install -r requirements.txt
 ```
 
-If building `nmslib` failed during installation, try manually installing the library with `conda install -c conda-forge nmslib` and following it up with `pip install -r requirements.txt` again.
-
-### Install vLLM or SGLang (Optional)
+### Install vLLM, SGLang, or TensorRT-LLM (Optional)
 
 #### vLLM
 
 ```bash
-pip install rank-llm[vllm]  # pip installation
-pip install -e .[vllm]      # or local installation for development
+pip install -e .[vllm]      # local installation for development
+pip install rank-llm[vllm]  # or pip installation
```

Comment:
This may not be necessary, but I put local installation ahead of PyPI. We are constantly updating RankLLM so PyPI may not always be up to date.